server/cli/flags_test.go

```diff
@@ -35,3 +36,31 @@ func TestStdFlagToPflag(t *testing.T) {
 		}
 	})
 }
+
+func TestNoLinkTesting(t *testing.T) {
+	imports := make(map[string]struct{})
+
+	var addImports func(string)
+	addImports = func(root string) {
+		pkg, err := build.Import(root, "", 0)
+		if err != nil {
+			t.Fatal(err)
+		}
+
+		for _, imp := range pkg.Imports {
+			// https: //github.com/golang/tools/blob/master/refactor/importgraph/graph.go#L115
+			if imp == "C" {
+				continue // "C" is fake
+			}
+			if _, ok := imports[imp]; !ok {
+				imports[imp] = struct{}{}
+				addImports(imp)
+			}
+		}
+	}
+
+	addImports("github.com/cockroachdb/cockroach")
```

Comment:
cool test. Could you add the "securitytest" package here as well? That's the one that embeds the test certs and we don't want it in the main binary. There's a build tag for that, but it could still happen with cached pkg files.